src/main/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetails.java

```diff
@@ -36,6 +36,7 @@
     private List<String> msqChoices;
     private boolean otherEnabled;
     private FeedbackParticipantType generateOptionsFor;
+    private boolean containsNonEmptyResponse;
```

Comment:
Are you sure you should be promoting `containsNonEmptyResponse` to an instance-level variable?